### DISCUSSION TITLE
Install extensions

### DISCUSSION THREAD

**Discussion by toerob** [toerob/vscode-tads3tools#26] [Ideas]
Is the install extension feature ever used? I have been sloppy in documenting it, and I wonder if just having it as a command will make it forgotten. So happy
to receive suggestions in how to front it more.

I also wonder if the ifarchive extension contribution folder is a bit too restrictive. I’m think about adding a new extension repository that holds a property file pointing out which file from which library is necessary to add to the makefile) so that it would be possible to make a complete installation. This instead of just downloading and unzipping folders to the game folder.

**Comment by johnnywz00** [comment]
It seems like it could be handy to have that feature if you're doing multiple projects from scratch in VSCode. I haven't used the feature mainly because my project is already 2 years under way and I already have all the extensions installed that I need...
